src/main/java/net/dries007/tfc/objects/blocks/BlocksTFC.java

```diff
@@ -297,6 +431,37 @@ public static void registerBlocks(RegistryEvent.Register<Block> event)
         normalItemBlocks.add(new ItemBlockTFC(register(r, "sea_ice", new BlockIceTFC(FluidsTFC.SALT_WATER.get()), CT_MISC)));
 
         normalItemBlocks.add(new ItemBlockLargeVessel(register(r, "ceramics/fired/large_vessel", new BlockLargeVessel(), CT_POTTERY)));
+        {
+            Builder<ItemBlock> raw = ImmutableList.builder();
```

Comment:
There's no need for the separate builders here. All the registration calls can be be adjusted to add the block to the respective map:
```java
// Inside each EnumDyeColor loop
BlockDecorativeStone block = new BlockDecorativeStone( ... );
normalItemBlocks.add(new ItemBlockTFC(register(r, "alabaster/whatever" + dyeColor.getName(), block)));
```